tests/test_tldr.py

```diff
@@ -27,7 +27,7 @@ def test_error_message():
     with mock.patch("sys.argv", ["tldr", "73eb6f19cd6f"]):
         with pytest.raises(SystemExit) as pytest_wrapped_e:
             tldr.main()
-        correct_output = "`73eb6f19cd6f` documentation is not available. If you want to contribute it, feel free to send a pull request to: https://github.com/tldr-pages/tldr." # noqa
+        correct_output = "`73eb6f19cd6f` documentation is not available. If you want to contribute it, feel free to send a pull request to: https://github.com/tldr-pages/tldr" # noqa
```

Comment:
Not the period 😢

Reply:
It will probably be readded, I was just testing. I can't find the issue which is breaking the test.